docs/_website/src/theme/Navbar/index.js

```diff
@@ -14,6 +14,28 @@ import useLogo from '@theme/hooks/useLogo';
 
 import styles from './styles.module.css';
 
+// items defined here instead of config so they can have an associated icon
+var items = [
```

Comment:
It looks like the navbar is now missing the docs, github, and community links

Reply:
fixed, ptal